docs/Tutorials/zot-registry-tutorial.mdx

```diff
@@ -0,0 +1,236 @@
+🔎  Verifying Kubernetes components...
+    ▪ Using image gcr.io/k8s-minikube/storage-provisioner:v5
+🌟  Enabled addons: storage-provisioner, default-storageclass
+🏄  Done! kubectl is now configured to use "minikube" cluster and "default" namespace by default
+```
+
+**Step 2: Specify a remote repository that contains the Helm charts for zot.**
+
+```
+helm repo add project-zot http://zotregistry.io/helm-charts
+```
+
+```
+"project-zot" has been added to your repositories
+```
+
+**Step 3: Update to the latest information of available charts from the chart repository, as shown below:**
+
+```
+helm repo update project-zot
+```
+
+```
+Hang tight while we grab the latest from your chart repositories...
+...Successfully got an update from the "project-zot" chart repository
+Update Complete. ⎈Happy Helming!⎈
+```
+
+You can use `helm show all project-zot/zot` to view the default information of the project. 
+
+To follow this tutorial, we do not need to make any changes to the default values. But, if you would like to see the configurable settings, run `helm show values project-zot/zot`. You can find more information on how to customize the Helm chart using `set` or a file through the [Helm documentation](https://helm.sh/docs/).
+
+**Step 4: Install zot**
+
+```
+helm install zot project-zot/zot
+```
+
+If you have successfully installed zot, you can expect a similar output:
+
+```
+NAME: zot
+LAST DEPLOYED: Sat May 20 15:50:22 2023
+NAMESPACE: default
+STATUS: deployed
+REVISION: 1
+NOTES:
+Get the application URL by running these commands:
+  export NODE_PORT=$(kubectl get --namespace default -o jsonpath="{.spec.ports[0].nodePort}" services zot)
+  export NODE_IP=$(kubectl get nodes --namespace default -o jsonpath="{.items[0].status.addresses[0].address}")
+  echo http://$NODE_IP:$NODE_PORT
+```
+
+Step 5: To verify the installation
+
+```
+export NODE_PORT=$(kubectl get --namespace default -o jsonpath="{.spec.ports[0].nodePort}" services zot)
+export NODE_IP=$(kubectl get nodes --namespace default -o jsonpath="{.items[0].status.addresses[0].address}")
+echo http://$NODE_IP:$NODE_PORT
+```
+
+Your output may be different but it should look like this:
+```
+http://192.168.49.2:30276
+```
+
+If you redirect to the above link, the screen will show the following:
+
+![](https://hackmd.io/_uploads/SJT67IYH3.png)
+
+
+### Phase 2: Distribution of OCI Artifacts to Zot Registry Using Oras CLI
+
+Please open a new terminal window for the next phase.
+
+**Note:** If you have used method 1 for the installation, you will need to replace the IP address and port with `localhost:5000`.
+
+**Step 1: Create a sample file** 
+
+```
+echo "hello world" > artifact.txt
+```
+
+**Step 2: Authenticate with zot registry**
+
+It is to be noted that in case of zot, authentication is disabled by default. Still, if you wish to authenticate, you can either use `docker login` or [`oras login`](https://oras.land/docs/CLI_Reference/oras_login) to do so.
```

Comment:
```suggestion
It is to be noted that in case of zot, authentication is disabled by default. Still, if you wish to authenticate, you can use [`oras login`](https://oras.land/docs/CLI_Reference/oras_login) to do so.
```
This is an oras tutorial, might as well cover the oras way